apps/blockchain/test/blockchain/state_test.exs

```diff
@@ -10,2365 +10,105 @@ defmodule Blockchain.StateTest do
-    "stRandom/randomStatetest64.json",
-    "stRandom/randomStatetest66.json",
-    "stRandom/randomStatetest67.json",
-    "stRandom/randomStatetest69.json",
-    "stRandom/randomStatetest7.json",
-    "stRandom/randomStatetest72.json",
-    "stRandom/randomStatetest73.json",
-    "stRandom/randomStatetest74.json",
-    "stRandom/randomStatetest75.json",
-    "stRandom/randomStatetest77.json",
-    "stRandom/randomStatetest78.json",
-    "stRandom/randomStatetest80.json",
-    "stRandom/randomStatetest81.json",
-    "stRandom/randomStatetest82.json",
-    "stRandom/randomStatetest83.json",
-    "stRandom/randomStatetest84.json",
-    "stRandom/randomStatetest85.json",
-    "stRandom/randomStatetest87.json",
-    "stRandom/randomStatetest88.json",
-    "stRandom/randomStatetest89.json",
-    "stRandom/randomStatetest9.json",
-    "stRandom/randomStatetest90.json",
-    "stRandom/randomStatetest92.json",
-    "stRandom/randomStatetest94.json",
-    "stRandom/randomStatetest95.json",
-    "stRandom/randomStatetest96.json",
-    "stRandom/randomStatetest97.json",
-    "stRandom/randomStatetest98.json",
-    # "stRandom2/201503110226PYTHON_DUP6.json",
-    "stRandom2/randomStatetest.json",
-    "stRandom2/randomStatetest384.json",
-    "stRandom2/randomStatetest385.json",
-    "stRandom2/randomStatetest386.json",
-    "stRandom2/randomStatetest387.json",
-    "stRandom2/randomStatetest388.json",
-    "stRandom2/randomStatetest389.json",
-    "stRandom2/randomStatetest391.json",
-    "stRandom2/randomStatetest393.json",
-    "stRandom2/randomStatetest395.json",
-    "stRandom2/randomStatetest396.json",
-    "stRandom2/randomStatetest397.json",
-    "stRandom2/randomStatetest398.json",
-    "stRandom2/randomStatetest399.json",
-    "stRandom2/randomStatetest401.json",
-    "stRandom2/randomStatetest402.json",
-    "stRandom2/randomStatetest404.json",
-    "stRandom2/randomStatetest405.json",
-    "stRandom2/randomStatetest406.json",
-    "stRandom2/randomStatetest407.json",
-    "stRandom2/randomStatetest408.json",
-    "stRandom2/randomStatetest409.json",
-    "stRandom2/randomStatetest410.json",
-    "stRandom2/randomStatetest411.json",
-    "stRandom2/randomStatetest412.json",
-    "stRandom2/randomStatetest413.json",
-    "stRandom2/randomStatetest414.json",
-    "stRandom2/randomStatetest415.json",
-    "stRandom2/randomStatetest416.json",
-    "stRandom2/randomStatetest417.json",
-    "stRandom2/randomStatetest418.json",
-    "stRandom2/randomStatetest419.json",
-    "stRandom2/randomStatetest420.json",
-    "stRandom2/randomStatetest421.json",
-    "stRandom2/randomStatetest422.json",
-    "stRandom2/randomStatetest423.json",
-    "stRandom2/randomStatetest424.json",
-    "stRandom2/randomStatetest425.json",
-    "stRandom2/randomStatetest426.json",
-    "stRandom2/randomStatetest428.json",
-    "stRandom2/randomStatetest429.json",
-    "stRandom2/randomStatetest430.json",
-    "stRandom2/randomStatetest433.json",
-    "stRandom2/r
```

Comment:
@masonforest wow :clap: . That's exactly what we need! Good work
 Can we use test name prefixed with its test group to easier find test? Also, can we add to `generate_state_tests` script  hardfork flag so it can print failing tests by hardfork